python/rikai/spark/sql/model.py

```diff
@@ -155,6 +156,19 @@ def model_type(self) -> "ModelType":
     def load_model(self) -> Any:
         """Load the model artifact specified in this spec"""
 
+    def load_id_to_label_fn(self) -> Optional[Callable]:
```

Comment:
shall we use `label_fn` for short?  we can make the typing to reflect the input/output types, i.e.

```Callable[[int], str]```.

Reply:
yeah, that's reasonable. I'll make that change